### PR TITLE
app-accessibility/brltty: bump to EAPI 6

### PR DESCRIPTION
**Pull request by hanetzer**
Also readjusted the epatch call to an eapply call, added
HTML_DOCS/einstalldocs as dohtml is deprecated on EAPI=6,
and adjusted the keepdir call to point at the correct directory.

Package-Manager: Portage-2.3.24, Repoman-2.3.6

### CODE REVIEW

File: app-accessibility/brltty/brltty-5.2-r1.ebuild [hunk 1]
```diff
@@ -166,18 +172,18 @@ src_install() {
 	mv "${D}"/${libdir}/*.a "${D}"/usr/${libdir}/
```
Comment (Moiman) [inline app-accessibility/brltty/brltty-5.2-r1.ebuild]: missing die

File: app-accessibility/brltty/brltty-5.2-r1.ebuild [hunk 2]
```diff
@@ -42,11 +42,17 @@ DEPEND="virtual/pkgconfig
...
+	"${FILESDIR}"/${PN}-5.2-respect-AR.patch
+	"${FILESDIR}"/${PN}-5.2-sysmacros.patch
+)
+
 src_prepare() {
-	epatch "${FILESDIR}"/${P}-fix-ldflags.patch \
-		"${FILESDIR}"/${P}-udev.patch \
-		"${FILESDIR}"/${P}-respect-AR.patch \
-		"${FILESDIR}"/${P}-sysmacros.patch
+	eapply "${PATCHES[@]}"
```
Comment (Moiman) [inline app-accessibility/brltty/brltty-5.2-r1.ebuild]: default

File: app-accessibility/brltty/brltty-5.2-r1.ebuild [hunk 3]
```diff
@@ -166,18 +172,18 @@ src_install() {
...
 	dohtml -r Manual-BRLTTY
 	if use doc; then
-		dohtml -r Manual-BrlAPI
 		dodoc BrlAPI-*.txt
+		einstalldocs
 	fi
 
-	keepdir /var/lib/brlapi
+	keepdir /var/lib/BrlAPI
 	rmdir "${D}/run/brltty"
```
Comment (Moiman) [inline app-accessibility/brltty/brltty-5.2-r1.ebuild]: missing dies

File: app-accessibility/brltty/brltty-5.2-r1.ebuild [hunk 4]
```diff
@@ -166,18 +172,18 @@ src_install() {
 	mv "${D}"/${libdir}/*.a "${D}"/usr/${libdir}/
 	gen_usr_ldscript libbrlapi.so
 
-	cd Documents
+	cd Documents || die
 	mv Manual-BRLTTY/English/BRLTTY.txt BRLTTY-en.txt
 	mv Manual-BRLTTY/French/BRLTTY.txt BRLTTY-fr.txt
 	mv Manual-BrlAPI/English/BrlAPI.txt BrlAPI-en.txt
```
Comment (Moiman) [inline app-accessibility/brltty/brltty-5.2-r1.ebuild]: missing dies

File: app-accessibility/brltty/brltty-5.2-r1.ebuild [hunk 5]
```diff
@@ -166,18 +172,18 @@ src_install() {
...
 	dodoc CONTRIBUTORS ChangeLog HISTORY README* TODO BRLTTY-*.txt
 	dohtml -r Manual-BRLTTY
```
Comment (Moiman) [inline app-accessibility/brltty/brltty-5.2-r1.ebuild]: dohtml is deprecated

File: app-accessibility/brltty/brltty-5.2-r1.ebuild (hunk 3 above)
Comment (hanetzer) [reply app-accessibility/brltty/brltty-5.2-r1.ebuild]: Didn't know you needed a die on `keepdir` and other ebuild builtins.

Comment (Moiman) [reply app-accessibility/brltty/brltty-5.2-r1.ebuild]: keepdir doesn't need die but rmdir probably needs.

File: app-accessibility/brltty/brltty-5.2-r1.ebuild (hunk 5 above)
Comment (hanetzer) [reply app-accessibility/brltty/brltty-5.2-r1.ebuild]: Must have missed that. Will fix.

File: app-accessibility/brltty/brltty-5.2-r1.ebuild (hunk 3 above)
Comment (hanetzer) [reply app-accessibility/brltty/brltty-5.2-r1.ebuild]: Ah. Will fix it.

**Pull request closed without merging**